ytgov/travel-authorization
# Update Objective field

# Context

Update the field description to be more meaningful. Change from "To Meet with colleagues around project x" to "Objectives"
![image](https://github.com/icefoganalytics/travel-authorization/assets/89539008/bf64b7c3-1645-4b29-830b-733ed1601ad8)

# Screenshots

Old
![image](https://github.com/ytgov/travel-authorization/assets/23045206/0e914cfe-515b-4b04-a2ba-eda7bf76d284)

New
![image](https://github.com/ytgov/travel-authorization/assets/23045206/a715d8bb-51e6-4713-9f85-48b30cdb95cf)

# Testing Instructions

1. Boot the database and api services via `dev up`
2. Boot the web service via `dev up_web`.
3. Log in to the app at http://localhost:8080.
4. Go to the "My Travel Requests" page via the top nav dropdown.
5. Click on an existing travel request in the table, or create one via the "+ Travel Authorization" if one does not exist.
6. Check that the field label of the "Objectives" section is now "Objectives".
